VMT/VMT.md

```diff
@@ -0,0 +1,225 @@
+| Triaged       | Impact description has been submitted for review |
+| In Progress   | CVE has been requested                           |
+| Fix committed | Pre-KCSA has been communicated                   |
+| Fix released  | All patches have been merged                     |
+| Opinion       | Issue is likely a Class B/C/D, waiting for KCSN  |
+| Won't Fix     | Doesn't fit with the project plans, sorry        |
+| Invalid       | Class E and Z, not a bug or vulnerability. No further action to be taken
+
+# Extent of Disclosure
+
+The science of vulnerability management is somewhere around being able to assess impact and severity of a report, being able to design security patches, being an obsessive process-following perfectionist and respecting the rule of lesser disclosure.
+
+Lesser disclosure is about disclosing the vulnerability details to an increasing number of people over time, but only to the people that are necessary to reach the next step.
+The diagram above shows "disclosure extent" across the various steps of the process.
+
+Vulnerability reporters retain final control over the disclosure of their findings.
+If for some reason they are uncomfortable with our process, their choice of disclosure terms prevails.
+
+## Embargo exceptions
+
+> **FIXME** - I'm not sure I fully understand the intent of the below - I need to re-read
+> and check the original...
+
+To keep the embargo period short and effective, the VMT may choose to open Issues.
+Issues that take too much time to be fixed (e.g., more than 2 weeks) or Issues that require a complex patch are usually better solved in the open.
+
+> **FIXME** is the architecture committee the correct group to be consulted here?
+
+Whenever such a case occurs, the Kata Containers [Architecture Committee](https://github.com/kata-containers/community#architecture-committee) is subscribed to the Issue in order to discuss whether or not it's imperative to keep that particular bug private.
+
+## Downstream stakeholders
+
+Kata Containers as an upstream project is used in a number of distributions, products, private and public service offerings that are negatively affected by vulnerabilities.
+In the spirit of [responsible disclosure](https://en.wikipedia.org/wiki/Responsible_disclosure), this ecosystem, collectively known as the downstream stakeholders, needs to be warned in advance to be able to prepare patches and roll them out in a coordinated fashion on disclosure day.
+The embargo period is kept voluntarily small (3-5 business days), as a middle ground between keeping the vulnerability under cover for too long and not giving a chance to downstream stakeholders to react.
+
+If you are currently not a referenced stakeholder and think you should definitely be included on that email distribution list, please submit an email with a rationale to member(s) of the [VMT mailing list](mailto:kata-vmt@lists.katacontainers.io).
```

Comment:
If we go the route of having VMT individual member info as the contact process, I believe we can set up a list.katacontainers.io list to require a message in the subscription request. I'll look into this.